src/menu/menu_state.h

```diff
@@ -87,12 +90,17 @@ typedef struct {
         bool go_left;
         bool go_right;
         bool go_fast;
+        bool go_c_left;
+        bool go_c_right;
+        bool go_c_up;
+        bool go_c_down;
 
         bool enter;
         bool back;
         bool options;
         bool settings;
         bool lz_context;
+        bool l;
```

Comment:
any reason that this is different from `lz_context`, as the function is the same for the rest of the menu (was maybe a merge confiict).